lib/services/local/recording_encryption.go

```diff
@@ -105,18 +157,42 @@ func newRecordingEncryptionParser() *recordingEncryptionParser {
 }
 
 func (p *recordingEncryptionParser) parse(event backend.Event) (types.Resource, error) {
```

Comment:
I'm not sure how to feel about using the same parser for the separate kinds, especially since we decided not to go with `rotated_keys` as a subkind. What do you think about just adding a separate parser for rotated keys?